.babelrc

```diff
@@ -0,0 +1,12 @@
+{
+    "presets": [
```

Comment:
What the f**k is this ???
Sorry for not understanding WebPack :,D

Reply:
This is the official documentation https://babeljs.io/docs/en/babel-preset-env , but basically the preset are a plugin develop by third party to allow certain functionality to babel core, like and ad-hoc approach